src/libraries/System.Globalization/tests/IcuTests.cs

```diff
@@ -10,7 +10,9 @@ namespace System.Globalization.Tests
     public class IcuTests
     {
         private static bool IsIcuCompatiblePlatform => PlatformDetection.IsNotWindows ||
-                                                       PlatformDetection.IsWindows10Version1903OrGreater;
+                                                       PlatformDetection.IsWindows10Version1903OrGreater &&
+                                                        // Server core doesn't have icu.dll on SysWOW64
+                                                        !(PlatformDetection.IsWindowsServerCore && PlatformDetection.IsX86Process));
```

Comment:
@safern there's an additional closing brace:
```suggestion
                                                        !(PlatformDetection.IsWindowsServerCore && PlatformDetection.IsX86Process);
```